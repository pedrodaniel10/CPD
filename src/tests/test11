2 20 100000000 5