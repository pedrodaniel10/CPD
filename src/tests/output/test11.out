0.70 0.81
0.50 0.50
